import test from 'ava'

test('It should display the error page (Error 404)', t => {
  console.log('Error')
})
